src/adapters/sqlite/encodeQuery/index.js

```diff
@@ -151,6 +151,28 @@ const encodeAssociation: (TableName<any>) => AssociationArgs => string = mainTab
 const encodeJoin = (table: TableName<any>, associations: AssociationArgs[]): string =>
   associations.length ? associations.map(encodeAssociation(table)).join('') : ''
 
+const encodeOrderBy = (table: TableName<any>, sortBys) => {
+  if (sortBys.length === 0) {
+    return ''
+  }
+  const orderBys = sortBys.map(sortBy =>
+    `${encodeName(table)}.${encodeName(sortBy.sortColumn)} ${sortBy.sortOrder}`
```

Comment:
can you add an invariant to Q.sortBy to ensure that you can't put something other than asc/desc there?